ament_cpplint/ament_cpplint/cpplint.py

```diff
@@ -5882,10 +5878,6 @@ def CheckCStyleCast(filename, clean_lines, linenum, cast_type, pattern, error):
            remainder):
     return False
 
-  # Don't warn in C files about C-style casts
```

Comment:
Oh I see this `patch`  already existed but was not documented.

Reply:
For reference added 9 years ago. :smiley: https://github.com/ament/ament_lint/pull/32